README.md

```diff
@@ -52,6 +50,14 @@ See [the documentation](https://shopware-pwa-docs.netlify.com/#introduction-to-s
 npm install -g @shopware-pwa/cli
 ```
 
+::: tip Canary version  
+Currently, we're releasing a `canary` version per every push to `master` branch, so in order to have newest changes and fixes just install it like this:
```

Comment:
Maybe we should mention that this version may contain some issues?

Reply:
I thought about it, but in some cases, it's more stable than alpha release ;D
When we move to Beta stage we'll add that kind of warning, as safer then just use Beta